articles/virtual-machine-scale-sets/virtual-machine-scale-sets-upgrade-scale-set.md

```diff
@@ -303,6 +303,9 @@ To update existing VMs, you must do a "manual upgrade" of each existing VM. You
 
 - You can also use the language-specific [Azure SDKs](https://azure.microsoft.com/downloads/).
 
+> [!NOTE]
+> This action will manually upgrade the selected instance to the latest model. While upgrading, the instance may be restarted.
```

Comment:
Please be more specific. We do not use "this" language in notes. Specify exactly what you mean by "this", even if the note is right next to the action you are referring to. Thank you!

Reply:
Done. Replace "this" with the command name.